src/extension.ts

```diff
@@ -222,7 +227,32 @@ export function activate(context: vscode.ExtensionContext) {
                 vscode.window.showInformationMessage("The projects have been refreshed!");
             }
         })
+    }
+
+    async function delay(ms: number) {
+        return new Promise(resolve => setTimeout(resolve, ms));
+    }
 
+    function refreshProjectsByType(projectType: string, locator: CustomProjectLocator, projectProvider: ProjectProvider, showMessage?: boolean, forceRefresh?: boolean) {
+        vscode.window.withProgress({
+            location: vscode.ProgressLocation.Notification,
+            title: `Refreshing ${projectType} Projects`,
```

Comment:
The progress message has changed. Now it shows: "`Refreshing Projects: ${projectType}"`, so you should remove `${projectType}` from this message and add it in the next `progress.report({...` line.